### PR TITLE
docs: Add doc for archived pvc removal

### PR DESCRIPTION
**Pull request by vatsa287**
Signed-off-by: Shree Vatsa N <vatsa@kadalu.io>

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -103,6 +103,24 @@ Remove storage by specifying the storage name to be deleted,
 $ kubectl kadalu storage-remove storage-pool-1
 ```
 
+### Remove archived PVCs
+
+Remove all archived pvc's provisioned from the storage-pool/volume.
+
+```
+$ kubectl kadalu remove-archived-pv storage-pool-1
```
Comment (leelavg) [inline cli/README.md:111]: - enforce `--pool` arg as well to be consistent with `--pvc`

File: doc/storage-config-options.adoc [hunk 2]
```diff
@@ -221,6 +221,31 @@ $ kubectl kadalu storage-add storage-pool1 \
...
+----
+$ kubectl kadalu remove-archived-pv storage-pool-1 --pvc=archived-pvc-123-456-789
+----
+
+* Removing archived pvc(s) through Kadalu CSI provisioner:
+Exec into Kadalu Provisioner pod. Run the script 'remove_archived_pv.py' with similar arguments as Kadalu CLI.
++
+[source,console]
+----
+$ python remove_archived_pv.py storage-pool-1 --pvc=archived-pvc-123-456-789
```
Comment (leelavg) [inline doc/storage-config-options.adoc:248]: - maybe `cd /kadalu; python ... ` woul be more accurate

File: cli/README.md (hunk 1 above)
Comment (vatsa287) [reply cli/README.md:111]: Storage pool name is a positional argument here. Should I make it optional? Then what should be the default behaviour? All storage pools?

File: doc/storage-config-options.adoc (hunk 2 above)
Comment (vatsa287) [reply doc/storage-config-options.adoc:248]: Sure will add it.

Comment (vatsa287) [reply doc/storage-config-options.adoc:248]: Done

File: cli/README.md (hunk 1 above)
Comment (leelavg) [reply cli/README.md:111]: - how about no positional argument with no default behaviour, is that feasible?
- user should provide either `--pool` or `--pool` and `--pvc`

**Review by leelavg — APPROVED** [review]
/lgtm

**Review by amarts — APPROVED** [review]

**Pull request merged**